go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/cyberdelia/templates v0.0.0-20141128023046-ca7fffd4298c
 	github.com/getkin/kin-openapi v0.2.0
 	github.com/go-chi/chi v4.0.2+incompatible
+	github.com/gogo/protobuf v1.3.1
```

Comment:
Could you use github.com/golang/protobuf instead of gogo?

Reply:
talked offline, and will use a registry that allows custom response handler.